src/main/java/edu/harvard/iq/dataverse/util/SignpostingResources.java

```diff
@@ -0,0 +1,273 @@
+            jsonSetting = "{" +
+                    "\"indetifier-schema\": {\"ORCID\":\"https://orcid.org/\", \"ISNI\":\"https://isni.org/isni/\", \"ScopusID\":\"https://www.scopus.com/authid/detail.uri?authorId=\"}," +
+                    "\"license\": {\"CC0\":\"https://creativecommons.org/licenses/by/4.0/\", \"MIT\": \"https://url\", \"APACHE\":\"https://url\"}," +
+                    "\"cite-as\": {\"doi\":\"https://citation.crosscite.org/format?style=bibtex&doi=\", \"type\":\"application/vnd.datacite.datacite+json\"}," +
+                    "\"useDefaultFileType\": true," +
+                    "\"defaultFileTypeValue\": \"https://schema.org/Dataset\"" +
+                    "}";
+        }
+        JsonReader jsonReader = Json.createReader(new StringReader(jsonSetting));
+        JsonObject spJsonSetting = jsonReader.readObject();
+        jsonReader.close();
+        idschemaJsonObj = spJsonSetting.getJsonObject("indetifier-schema");
+        licJsonObj = spJsonSetting.getJsonObject("license");
+        citeAsJsonObj = spJsonSetting.getJsonObject("cite-as");
+        useDefaultFileType = spJsonSetting.getBoolean("useDefaultFileType", true);
+        defaultFileTypeValue = spJsonSetting.getString("defaultFileTypeValue", "https://schema.org/Dataset");
+    }
+
+    /**
+     * Get identifier schema for each author
+     *
+     * Author may have identifiers from different providers
+     * ORCID: the url format is https://orcid.org/:id
+     * ISNI: the url format is https://isni.org/isni/:id
+     * ScopusID: the url format is https://www.scopus.com/authid/detail.uri?authorId=:id
+     * VIAF: the url format is http://viaf.org/viaf/:id
+     *
+     * For example:
+     *      if author has VIAF
+    *       Link: <http://viaf.org/viaf/:id/>; rel="author"
+     *
+     * @param datasetAuthors
+     * @return
+     */
+    private String getIdentifierSchema(List<DatasetAuthor> datasetAuthors) {
+        StringBuilder sb = new StringBuilder();
+        String identifierSchema = "";
+
+        for (DatasetAuthor da:datasetAuthors){
+            if (da.getIdValue() != null && !da.getIdValue().isEmpty()) {
+                sb.append("<").
+                        append(idschemaJsonObj.getString(da.getIdType())).
+                        append(da.getIdValue()).
+                        append(">;rel=\"author\"");
+                if (identifierSchema == "") {
+                    identifierSchema = sb.toString();
+                } else {
+                    identifierSchema = String.join(",", identifierSchema, sb.toString());
+                }
+            }
+        }
+        return identifierSchema;
+    }
+
+    public String getLinks(){
+        // TODO: individual value should be ignored if it's empty
+        // list of the strings going to be returned
+        List<String> valueList = new LinkedList<>();
+        Dataset ds = workingDatasetVersion.getDataset();
+
+        // signposting identifierSchema
```

Comment:
Remove comments that add little to no information. In this case the only extra information is that is has to do with "signposting", but we don't have to tell that every other line.